src/core/device.c

```diff
@@ -90,6 +92,45 @@ static int device_set_sysfs(Device *d, const char *sysfs) {
         return 0;
 }
 
+static void io_cost_qos_reset(IOCostQos *io_cost_qos) {
+        assert(io_cost_qos);
+
+        io_cost_qos->enabled = -1;
+        io_cost_qos->ctrl = _IO_COST_CTRL_INVALID;
+        io_cost_qos->read_latency_percentile = 0;
+        io_cost_qos->read_latency_threshold = 0;
+        io_cost_qos->write_latency_percentile= 0;
+        io_cost_qos->write_latency_threshold= 0;
+        io_cost_qos->min= 0;
+        io_cost_qos->max= 0;
+        /* Lowest possible value for min and max is 1 */
```

Comment:
This comment doesn't seem to match the setting. I think you mean to say something like if 0 is supplied the kernel might still set it to something else?